src/components/Header.js

```diff
@@ -166,24 +171,11 @@ function Header(){
                     </div>
                     <div className="col-1 text-center p-0">
                         <div className="text-primary py-3 py-xl-5 font-size-1"
-                        // onMouseDown={() => {
-                        //     holdTimeoutRef.current = setTimeout(() => handleAdminLongPress(), 500);
-                        //   }}
-                        //   onMouseUp={() => clearTimeout(holdTimeoutRef.current)}
-                        //   onTouchStart={() => {
-                        //     holdTimeoutRef.current = setTimeout(() => handleAdminLongPress(), 500);
-                        //   }}
-                        //   onTouchEnd={() => clearTimeout(holdTimeoutRef.current)}
-                        //   onMouseLeave={() => clearTimeout(holdTimeoutRef.current)}
-                          onMouseDown={() => {
-                            holdTimeoutRef.current = setTimeout(() => handleAdminLongPress(), 500);
-                          }}
-                          onMouseUp={() => clearTimeout(holdTimeoutRef.current)}
-                          onTouchStart={() => {
-                            holdTimeoutRef.current = setTimeout(() => handleAdminLongPress(), 500);
-                          }}
-                          onTouchEnd={() => clearTimeout(holdTimeoutRef.current)}
-                          onMouseLeave={() => clearTimeout(holdTimeoutRef.current)}
+                            onMouseDown={handleAdminLongPress}
+                            onMouseUp={() => window.CrComLib.publishEvent('b', '120', false)}
+                            onTouchStart={handleAdminLongPress}
+                            onTouchEnd={() => window.CrComLib.publishEvent('b', '120', false)}
```

Comment:
quick note: `onTouchEnd` might not trigger if the user taps on the button/div and then slides their finger outside of the bounding rect of the button/div, thus not triggering the onTouchEnd event on the button/div (!). This can leave a button with a `pressed state` while the user has no fingers touching the panel. If this causes problems, consider listening for onTouchEnd at a higher level element (even at the body) to ensure that onTouchEnd it is being tracked.

Reply:
I haven't gotten to solving this bug but it's on the triage list and I'll be tackling it soon.